blackbox/docs/sql/aggregation.txt

```diff
@@ -205,8 +205,13 @@ Some Examples::
 =======
 
 The ``sum`` aggregation function returns the sum of all the values in a column
```

Comment:
pls change to ... returns the sum of a set of numeric input values that are not ...

and remove the last sentence. 

background: the implication that this only works on columns is wrong: select sum(1) also works